src/app/(proper_react)/(redesign)/Shell/Shell.stories.ts

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Session } from "next-auth";
+import { SerializedSubscriber } from "../../../../next-auth";
+
+import type { Meta, StoryObj } from "@storybook/react";
+import { Shell } from "./Shell";
+import { getL10n } from "../../../functions/l10n/storybookAndJest";
+
+function createUser(): Session["user"] {
```

Comment:
Question: why is this wrapped in a function? :)

Reply:
Move it directly into mockedSession in [0cf43e5](https://github.com/mozilla/blurts-server/pull/5671/commits/0cf43e532749c2b333f0409678eda3c49bf30ada).